docs/UserGuide.md

```diff
@@ -29,6 +29,7 @@ Using NUS Classes can get your contact management tasks done faster than traditi
         * [Mark a task as done](https://github.com/AY2122S2-CS2103T-T12-4/tp/blob/master/docs/UserGuide.md#mark-a-task-as-done-mark)
         * [Unmark a task as not done](https://github.com/AY2122S2-CS2103T-T12-4/tp/blob/master/docs/UserGuide.md#unmark-a-task-as-not-done-unmark)
         * [Deleting tasks](https://github.com/AY2122S2-CS2103T-T12-4/tp/blob/master/docs/UserGuide.md#deleting-tasks-deletet)
+        * [Generating emails of all contacts related to a task](https://github.com/AY2122S2-CS2103T-T12-4/tp/blob/master/docs/UserGuide.md#gen)
```

Comment:
Sorry I realised we don't need the direct link, can you change this to simply #gen ? Then it'll point to the local version

Reply:
okie donezo